### PR TITLE
fix: add missing menu items  for sequencer tier

### PR DESCRIPTION
**Pull request by evilpeach**

### CODE REVIEW

File: src/lib/layout/mobile/NavDrawer.tsx [hunk 1]
```diff
@@ -43,37 +48,63 @@ export const NavDrawer = () => {
...
+          wasmConfig.enabled,
+          moveConfig.enabled,
+          nftConfig.enabled
+        );
+        break;
+      case "sequencer":
+        navMenuTmp = getNavDrawerSequencer(
+          govConfig.enabled,
+          wasmConfig.enabled,
+          moveConfig.enabled
```
Comment (songwongtp) [inline src/lib/layout/mobile/NavDrawer.tsx:66]: Dont we also need nft enabled?

**Review by songwongtp — APPROVED** [review]

File: src/lib/layout/mobile/utils.ts [hunk 2]
```diff
@@ -52,6 +53,72 @@ export const getNavDrawerLite = (
...
+              slug: "/interact-contract",
+              icon: "query" as IconKeys,
+            },
+          ]
+        : []),
+      ...(isMove
+        ? [
+            {
+              name: "0x1 Page",
+              slug: "/account/0x1",
```
Comment (songwongtp) [inline src/lib/layout/mobile/utils.ts:108]: accounts

File: src/lib/layout/mobile/NavDrawer.tsx (hunk 1 above)
Comment (evilpeach) [reply src/lib/layout/mobile/NavDrawer.tsx:66]: There is no collection list page in sequencer tier.

File: src/lib/layout/mobile/utils.ts (hunk 2 above)
Comment (evilpeach) [reply src/lib/layout/mobile/utils.ts:108]: Noted

**Pull request merged**